Tranlate to German

AKingUltra opened the pull request:
Translation into German, one question remains: Do you want formal or informal language? This translation is in the formal style, please let me know if informal is preferred.

AKingUltra committed:
Create messages.json

Translation into German, one question remains: Do want formal or informal? This translation is in the formal style, please let me know if informal is preferred.

tmashuang commented:
We have made added more to the `messages.json` file, please update and add.

kumavis commented:
@AKingUltra we've been preferring more informal language
also if you want to use quotations (`"`) in translations, you'll need to escape them (`\"`).

kumavis commented on app/_locales/de/messages.json:
```diff
@@ -0,0 +1,609 @@
+    "description": "Wählen Sie diesen Dateityp um damit einen Account zu importieren"
+  },
+  "privateKeyWarning": {
+    "message": "Warnung: Niemals jemanden den Private Key mitteilen. Jeder der im Besitzt Ihres Private Keys ist, kann jegliches Guthaben Ihres Accounts stehlen.  
```
need to close the string `"`

kumavis commented on app/_locales/de/messages.json:
```diff
@@ -0,0 +1,609 @@
+  },
+  "toETHviaShapeShift": {
+    "message": "$1 an ETH via ShapeShift",
+    "description": "Das System wird den Einzahlungstyp im Beginn der Nachricht eintragen“
```
need to close the string with a normal quotation mark (`"`)

AKingUltra commented:
Hi kumavis. Alrighty, I'll change it tomorrow and will fix the mistakes. Cheerio!

kumavis commented on app/_locales/de/messages.json:
```diff
@@ -0,0 +1,609 @@
+    "message": "QR Code anzeigen"
+  },
+  "readdToken": {
+    "message": "Sie können diesen Token zukünftig wieder hinzufügen indem Sie in den Menüpunkt "Token hinzufügen" in den Einstellungen Ihres Accounts gehen."
```
need to escape quotes with the slash
```
\"Token hinzufügen\"
```

AKingUltra committed:
Update messages.json

Integrated the newly added translations and switched from formal to informal wording.

AKingUltra commented:
Hi again, 
I just checked out the Beta in German more thoroughly and I found a few missing translations, formatting and graphic errors. I don't know if this is the right place for this but I uploaded my findings and some screenshots.
[Errors.xlsx](https://github.com/MetaMask/metamask-extension/files/1859301/Errors.xlsx)
